src/translations/fr.json

```diff
@@ -10,7 +10,7 @@
     "deleteStudyError": "Vous ne pouvez pas supprimer une étude dont vous n'êtes pas propriétaire",
     "equipmentsSearchingError": "Une erreur est survenue lors de la recherche des équipements",
 
-    "renameStudyMsg": "Nouveau nom de l'étude : ",
+    "renameStudyMsg": "Nouveau nom de l'étude \u00a0:",
```

Comment:
Some of the unbreakable spaces are preceded by a regular space that may prevent unbreakable effect.